mlflow/server/js/src/common/components/EditableNote.tsx

```diff
@@ -233,6 +233,7 @@ function HTMLNoteContent(props: HTMLNoteContentProps) {
       <div className="note-view-text-area">
         <div className="note-view-preview note-editor-preview">
           <div
+            style={{ wordBreak: 'break-word' }}
```

Comment:
nit: use `css` instead
```suggestion
            css={{ wordBreak: 'break-word' }}
```